### PR TITLE
Kg   shodenna 

### PR DESCRIPTION
**Pull request by Keeppie**
Added text to Shodenna page with 9 buttons that lead to 9 other pages with text in them

### CODE REVIEW

File: Routes/MainStack.js [hunk 1]
```diff
@@ -15,6 +15,16 @@ import Marennya from "/screens/psychologichnaDopomoga/ekstrennaDopomoga/ekstrenn
...
+import Shodenna from "/screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/Shodenna";
+  import Shvidkii from "../screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Shvidkii";
+  import Spokiine from "../screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Spokiine";
+  import VnutrishniiSad from "../screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/VnutrishniiSad";
+  import VnutrishniiSeif from "../screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/VnutrishniiSeif";
+  import CheroBilaFoto from "../screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/CheroBilaFoto";
+  import ChernoBilaCadr from "../screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/ChernoBilaCadr";
+  import Tehnika from "../screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Tehnika";
+  import Kvitka from "../screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Kvitka";
+  import YaMau from "../screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/YaMau";
```
Comment (garys-primary) [inline Routes/MainStack.js]: please fix formatting (remove extra spaces) for all above lines

Comment (garys-primary) [reply Routes/MainStack.js]: Also make paths absolute (/screens instead of ../screens)

File: Routes/MainStack.js [hunk 2]
```diff
@@ -58,6 +68,19 @@ function MainStack() {
...
+      
+        <Stack.Screen name="Shvidkii" component={Shvidkii} />
+        <Stack.Screen name="Spokiine" component={Spokiine} />
+        <Stack.Screen name="VnutrishniiSad" component={VnutrishniiSad} />
+        <Stack.Screen name="VnutrishniiSeif" component={VnutrishniiSeif} />
+        <Stack.Screen name="CheroBilaFoto" component={CheroBilaFoto} />
+        <Stack.Screen name="ChernoBilaCadr" component={ChernoBilaCadr} />
+        <Stack.Screen name="Tehnika" component={Tehnika} />
+        <Stack.Screen name="Kvitka" component={Kvitka} />
+        <Stack.Screen name="YaMau" component={YaMau} />
```
Comment (garys-primary) [inline Routes/MainStack.js]: Formatting

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/Shodenna.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import Header from "/components/Header";
+import resolveStyles from "/styles/subpage";
+import ListItem from "/components/ListItem";
+
+
+export default function Shodenna({ navigation }) {
+  const { computeTheme } = React.useContext(ThemeContext);
+  const { computeFontSize } = React.useContext(FontSizeContext);
+  const styles = resolveStyles(computeTheme(), computeFontSize);
+  const navigateAway = () => {// eslint-disable-next-line no-console console.log("navigation");
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/Shodenna.js]: please remove this function

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/ChernoBilaCadr.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import Header from "/components/Header";
+import resolveStyles from "/styles/subpage";
+import ListItem from "/components/ListItem";
+
+export default function ChernoBilaCadr({ navigation }) {
+  const { computeTheme } = React.useContext(ThemeContext);
+  const { computeFontSize } = React.useContext(FontSizeContext);
+  const styles = resolveStyles(computeTheme(), computeFontSize);
+  const navigateAway = () => {// eslint-disable-next-line no-console console.log("navigation");
+ };
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/ChernoBilaCadr.js]: also remove function

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/CheroBilaFoto.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+CheroBilaFoto
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/CheroBilaFoto.js]: please remove this line

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Kvitka.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
+Kvitka
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Kvitka.js]: please remove this

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Shvidkii.js [hunk 7]
```diff
@@ -0,0 +1,104 @@
+Shvidkii
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Shvidkii.js]: please remove this line

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Shvidkii.js [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+          <Bullet symbol="•." style={styles.paragraph}>
+          Звіряємо смак – має бути сповнено відчуттям мужності та рішучості, містити аромат любові та глибокого миру. Може включати в залежності від місця і часу виготовлення також елементи тривоги, смутку, праведного гніву – але без гіркоти відчаю, присмаку паніки чи озлоблення. Часто містить відчуття радості та вдячності.
+          </Bullet>
+
+          <Bullet symbol="•" style={styles.paragraph}>
+          Якщо щось смакує не так – коригуємо:
+          </Bullet>
+
+----------SubBullet (needs space):
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Shvidkii.js]: please comment this out with //

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Shvidkii.js [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+          є відчай – додаємо надії;
+          </Bullet>
+          <Bullet symbol="•" style={styles.paragraph}>
+          паніку нейтралізуємо відвагою;
+          </Bullet>
+          <Bullet symbol="•" style={styles.paragraph}>
+          багато смутку і болю – потребуємо співчуття…
+          </Bullet>
+
+-----------End of SubBullet
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Shvidkii.js]: please comment this out

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Spokiine.js [hunk 10]
```diff
@@ -0,0 +1,46 @@
+Spokiine
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Spokiine.js]: please remove this line

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Tehnika.js [hunk 11]
```diff
@@ -0,0 +1,63 @@
+Tehnika
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/Tehnika.js]: please remove this line

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/VnutrishniiSad.js [hunk 12]
```diff
@@ -0,0 +1,79 @@
+VnutrishniiSad
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/VnutrishniiSad.js]: please remove this line

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/VnutrishniiSeif.js [hunk 13]
```diff
@@ -0,0 +1,44 @@
+VnutrishniiSeif
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/VnutrishniiSeif.js]: please remove

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/YaMau.js [hunk 14]
```diff
@@ -0,0 +1,65 @@
+YaMau
```
Comment (garys-primary) [inline screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/YaMau.js]: please remove

**Review by garys-primary — CHANGES_REQUESTED** [review]
Please see comments. Otherwise - great job

File: screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/ChernoBilaCadr.js (hunk 4 above)
Comment (Keeppie) [reply screens/psychologichnaDopomoga/dlyaSebe/dlyaSebeScreens/ShodennaFile/ChernoBilaCadr.js]: Everything? line 14 to 17th?

File: Routes/MainStack.js (hunk 1 above)
Comment (Keeppie) [reply Routes/MainStack.js]: Forgot about it =)

**Review by garys-primary — APPROVED** [review]
Good job!

**Pull request merged**